OgcApi.Net.MbTiles/MbTilesProvider.cs

```diff
@@ -51,22 +51,44 @@ public List<TileMatrixLimits> GetLimits(string collectionId)
                 using var connection = GetDbConnection(tileOptions.FileName);
                 connection.Open();
 
-                var command = GetDbCommand(@"SELECT zoom_level, MIN(tile_column), MAX(tile_column), MIN((1 << zoom_level) - 1 - tile_row), MAX((1 << zoom_level) - 1 - tile_row) FROM tiles GROUP BY zoom_level ORDER BY zoom_level", connection);
+                var checkMetadataCommand = GetDbCommand(@"SELECT name FROM sqlite_master WHERE type = 'table' AND name = 'metadata'", connection);
+                if (checkMetadataCommand.ExecuteScalar() == null) return null;
+                
+                var getParamCommand = GetDbCommand(@"SELECT Value FROM metadata WHERE name = $name", connection);
+                    
+                getParamCommand.Parameters.AddWithValue("$name", "minzoom");
+                if (!int.TryParse(getParamCommand.ExecuteScalar().ToString(), out int minZoom)) return null;
```

Comment:
Possible null reference exception. ToString cannot be called for null value. 
Use var instead of int